Modules/_cursesmodule.c

```diff
@@ -163,13 +163,9 @@ typedef chtype attr_t;           /* No attr_t type is available */
 
 /*[clinic input]
 module _curses
-class _curses.window "PyCursesWindowObject *" "&PyCursesWindow_Type"
+class _curses.window "PyCursesWindowObject *" "clinic_state()->PyCursesWindow_Type"
 [clinic start generated code]*/
-/*[clinic end generated code: output=da39a3ee5e6b4b0d input=43265c372c2887d6]*/
-
-/* Definition of exception curses.error */
-
-static PyObject *PyCursesError;
+/*[clinic end generated code: output=da39a3ee5e6b4b0d input=83369be6e20ef0da]*/
 
 /* Tells whether setupterm() has been called to initialise terminfo.  */
 static int initialised_setupterm = FALSE;
```

Comment:
These would also have to be moved to module state?

Reply:
Done. Not sure if I did it the best way though.

Reply:
Sure I didn't. At least exposing parts of `_curses` module state to C API doesn't look safe for me.